cmd/init.go

```diff
@@ -0,0 +1,556 @@
+package cmd
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"os"
+	"sort"
+	"strconv"
+	"strings"
+
+	"github.com/amp-labs/cli/flags"
+	"github.com/amp-labs/cli/logger"
+	"github.com/amp-labs/cli/openapi"
+	"github.com/amp-labs/cli/request"
+	"github.com/manifoldco/promptui"
+	"github.com/spf13/cobra"
+	"sigs.k8s.io/yaml"
+)
+
+var ErrBadInput = errors.New("bad input")
+
+const (
+	NumProvidersShown = 10
+	YamlFileMode      = 0o644
+)
+
+func nonEmpty(fieldName string) func(string) error {
+	return func(s string) error {
+		if s == "" {
+			return fmt.Errorf("%w: %s cannot be empty", ErrBadInput, fieldName)
+		}
+
+		return nil
+	}
+}
+
+var initCmd = &cobra.Command{ //nolint:gochecknoglobals
+	Use:   "init",
+	Short: "create a new Ampersand project",
+	Long:  "Create a new Ampersand project.",
```

Comment:
Can you hide this command for now? I want to play with it more and tweak the UX more before releasing.